backend/locale/he.po

```diff
@@ -0,0 +1,821 @@
+msgstr "תוכנה: סוג השימוש"
+
+msgid "computer-knowledge-category"
+msgstr "ידע במחשבים"
+
+msgid "linux-knowledge-category"
+msgstr "לינוקס ואני"
+
+msgid "presets-category"
+msgstr "התקנה: מערכי הגדרות"
+
+msgid "hardware-category"
+msgstr "תמיכה בחומרה"
+
+msgid "help-category"
+msgstr "מקור קבלת עזרה"
+
+msgid "ux-concept-category"
+msgstr "חוויית המשתמש"
+
+msgid "price-category"
+msgstr "הפצות: מחיר"
+
+msgid "scope-category"
+msgstr "הפצות: היקף יישומים"
+
+msgid "ideology-category"
+msgstr "הפצות: אידאולוגיה"
+
+msgid "privacy-category"
+msgstr "הפצות: פרטיות"
+
+msgid "software-admin-category"
+msgstr "ניהול מערכת"
+
+msgid "software-updates-category"
+msgstr "תוכנה: עדכונים"
+
+msgid "special-features-category"
+msgstr "תוכנה: תכונות מיוחדות"
+
+msgid "recommendation-category"
+msgstr "הצגת התוצאות שלי"
+
+msgid "welcome-text-title"
+msgstr "ברוך בואך אל distrochooser.de!"
+
+msgid "welcome-text"
+msgstr "ברוך בואך! שאלון זה יעזור לך לבחור הפצת לינוקס מתאימה."
+
+msgid "welcome-text-skip"
+msgstr "תמיד אפשר לדלג על שאלות"
+
+msgid "welcome-text-result-get"
+msgstr "תמיד אפשר ללחוץ על \"ההמלצה שלך\" כדי לקבל את התוצאות"
+
+msgid "welcome-text-order"
+msgstr "אפשר לענות לא לפי הסדר"
+
+msgid "welcome-text-additional-infos"
+msgstr "אפשר לקבל מידע נוסף בעזרת לחיצה על סמל סימן השאלה"
+
+msgid "welcome-text-remove"
+msgstr "תמיד אפשר למחוק תשובות"
+
+msgid "welcome-text-feedback"
+msgstr "נא לדרג את התוצאות שלך, לפי מידת התאמתן לצרכים שלך."
+
+msgid "welcome-text-importance"
+msgstr "אפשר לסמן תשובות כחשובות כדי לתת להן יותר משקל"
+
+msgid "locale-link-en"
+msgstr "הטקסט הזה זמין באנגלית"
+
+msgid "locale-link-de"
+msgstr "שאלון זה זמין גם בגרמנית"
+
+msgid "start-test"
+msgstr "להתחלת השאלון"
+
+msgid "prev-question"
+msgstr "אחורה"
+
+msgid "next-question"
+msgstr "קדימה"
+
+msgid "no-results"
+msgstr "סליחה, איבדתי אותך"
+
+msgid "about-header"
+msgstr "על distrochooser.de"
+
+msgid "vote-reminder"
+msgstr "אהבת את ההפצה? אפשר לסמן אותה בלב!"
+
+msgid "vote-reminder-negative"
+msgstr "לא אהבת את ההפצה? אפשר לסמן אותה באגודל כלפי מטה!"
+
+msgid "about-intro-text"
+msgstr ""
+"‏distrochooser.de מנסה לעזור למתחילים בלינוקס לבחור הפצה לינוקס המתאימה"
+" לצורכיהם. <br> התוצאות שניתנו הן הצעות המבוססות על התשובות שלך. לפעמים"
+" התוצאות עשויות להיות לא מתאימות בגלל אינטראקציות מורכבות בין תשובות."
+
+msgid "about-intro-test-count"
+msgstr "מאז ה־14 ביוני 2014 distrochooser.de עבר %tests% בדיקות."
+
+msgid "about-licenses"
+msgstr "‏Distrochooser מוגן בשני רישיונות."
+
+msgid "privacy-header"
+msgstr "הגנת נתונים"
+
+msgid "imprint-header"
+msgstr "הערות"
+
+msgid "about-licenses-source"
+msgstr "קוד המקור"
+
+msgid "about-licenses-source-text"
+msgstr ""
+"קוד המקור מוגן ברישיון שכתוב בקובץ \"LICENSE\", אשר מאוחסן בתוך המאגר אליו"
+" ניתן הקישור."
+
+msgid "about-licenses-database"
+msgstr "מסד הנתונים"
+
+msgid "about-licenses-database-text"
+msgstr ""
+"The underlying data, including the question/answer structure, the result"
```

Comment:
If your comment was telated to this translation, you can also aim to translate for following sentence:

"The underlying data, including the question/answer structure, the result"
" information and statistics, are proprietary. You may use them only with our prior"
" permission. The description texts and logos of the various distributions are"
" subject to the licences of the respective copyright holders."